docs/data/pages.ts

```diff
@@ -140,9 +140,10 @@ const pages: MuiPage[] = [
           {
             pathname: '/x/react-data-grid/server-side-data-group',
             title: 'Server-side data',
+            unstable: true,
             plan: 'pro',
             children: [
-              { pathname: '/x/react-data-grid/server-side-data', title: 'Overview' },
+              { pathname: '/x/react-data-grid/server-side-data', title: 'Overview', plan: 'pro' },
```

Comment:
I would lobby to have this in the MIT plan, to make it easier to adopt the data grid against other options, and to make it easier to access Pro features, but ok, we can always easy change this.

Reply:
I'll be happy to revisit the [decision](https://www.notion.so/Data-source-proposal-db58b8b749154420a407f231c5528c1d?d=69c1ae33f04a4a37a49b8c726bf3451b&pvs=4#2a2daff41fdd4b5dbf7d3c5baa0fbad7) around the licensing. I can personally see no harm in making the basic data source an MIT feature, as an alternative to the conventional `XXXMode='server'` dx. Agree with the plus to make the adaptability easier.

I wanted to do that a bit later once the Pro/Premium counterparts gain a considerable user base, to be separately announced as a strengthening point for the MIT users as we did with the [column resizing](https://mui.com/blog/mui-x-v7/#column-resizing-on-the-community-plan). This might be good for marketing.

The pre-release phase could also be a good time though, since we can announce on the stable release.
What do you think @joserodolfofreitas @cherniavskii?

Reply:
@MBilalShafi Working on this with a handful of people, until they love it sounds great. So Pro only for the time being 👍. 

Once we get this kind of feedback https://mui.zendesk.com/agent/tickets/22713

> It was easier to implement loading data with offset to parallelize the load and get more performance.

We would know it's time to make it stable, and MIT available at the same time?